style.css

```diff
@@ -277,3 +277,1510 @@ li {
 .shape11:hover {
   background: none;
 }
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  height: min-content;
+  position: absolute;
+  width: 93.5%;
+  top: 759px;
+  z-index: 3;
+}
+
+.container1 {
```

Comment:
- [ ] Kindly use one class to style all your projects since the design is the same to avoid code repetition. Currently you have each project with its class, leading to code repetition.

Reply:
Hi i dont get this @danmainah